eng/codegen_to_sdk_config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "init": {
+      "initScript": {
+        "path": ".scripts/automation_init.sh"
+      }
+    },
+  "generateAndBuild": {
+      "generateAndBuildScript": {
+        "path": ".scripts/automation_generate.sh",
+        "stdout": {
+          "storeLogByFilter": "Error: |error |throw|exception"
```

Comment:
I know nothing about this field, but is the space after `error` in the value intended?

Reply:
This field is usd by sdk generation pipeline, instead of sdk automation pipeline in swagger PR. The space is intended.